doc/modules/mixture.rst

```diff
@@ -68,8 +68,9 @@ full covariance.
     * See :ref:`sphx_glr_auto_examples_mixture_plot_gmm_pdf.py` for an example on plotting the
       density estimation.
 
-Pros and cons of class :class:`GaussianMixture`
------------------------------------------------
+|details-start|
+**Pros and cons of class GaussianMixture**
+|details-split|
 
 Pros
 ....
```

Comment:
I am thinking that maybe it's better to use a `.. topic::` directive whenever we mention pros and cons
```suggestion
.. topic:: Pros:
```

Remember you will need to add an indentation to the paragraphs concerned by the directive.

Reply:
Fixed in the latest commit